_episodes/02-working-with-environments.md

```diff
@@ -350,6 +352,34 @@ Running `ls` (linux) / `dir` (Windows) on your anaconda `envs/` directory will l
 > previously installed Conda environments.
 {: .callout}
 
+## Workspace for Conda environments
```

Comment:
Why include this here as opposed to the `setup.md` file? Am not opposed to including this in the boy of the lessons but if we do so I think it should be near the beginning of episode 1.

Reply:
See answer above for setup.